analytics-webapps/src/main/webapp/WEB-INF/conf/analytics/portal/portal/global/pages.xml

```diff
@@ -602,7 +602,16 @@
                    "columns":[
                       {
                          "title":"analytics.creationDate",
-                         "spaceField":"createdTime",
+                         "previousPeriod":false,
+                         "valueAggregation":{
+                               "aggregation":{
+                                 "sortDirection":"desc",
+                                 "field":"spaceCreatedTime",
+                                 "type":"MAX",
+                               },
+                               "periodIndependent":false
```

Comment:
```suggestion
                               "periodIndependent":true
```
This flag should be period independent, else, if the user selects a range where the `spaceCreatedTime` doesn't exist, the column will not display anything